README.md

```diff
@@ -13,7 +13,7 @@ I used this kind of hot reloading while developing my game [CAT & ONION](https:/
 If you are on Linux / macOS: Below, replace `.bat` with `.sh` and `.exe` with `.bin`.
 
 1. Run `build_hot_reload.bat` to create `game.exe` and `game.dll`. Note: It expects odin compiler to be part of your PATH environment variable.
-2. Run `game.exe`, leave it running. Note: On Windows you have to copy `raylib.dll` from `your_odin_compiler/vendor/raylib/windows` into this folder.
+2. Run `game.exe`, leave it running. Note: On Windows, if you installed odin through scoop or set an environment variable named `ODIN_ROOT` to the root of your odin install, the script will work out of the box. Otherwise you have to copy `raylib.dll` from `your_odin_compiler/vendor/raylib/windows` into this folder.
```

Comment:
Thanks for the PR. I don't understand this documentation change. In `build_hot_reload.bat` you use `odin root` to tell you where the root of the odin compiler is. So you do not need `ODIN_ROOT` env var. You just need Odin to be in the path.

Reply:
Oh, yeah, I first based it off of scoop and forgot to change the docs back. Sorry for that